ceviche/fdfd.py

```diff
@@ -191,6 +191,123 @@ def _solve_fn(self, eps_vec, entries_a, indices_a, Jz_vec):
         Hx_vec, Hy_vec = self._Ez_to_Hx_Hy(Ez_vec)
         return Hx_vec, Hy_vec, Ez_vec
 
+
+class fdfd_mf_ez(fdfd):
+    """ FDFD class for multifrequency linear Ez polarization """
+
+    def __init__(self, omega, dL, eps_r, Omega, delta, phi, Nsb, npml, bloch_phases=None):
+        super().__init__(omega, dL, eps_r, npml, bloch_phases=bloch_phases)
+        self.Omega = Omega
+        self.delta = delta
+        self.phi = phi
+        self.Nsb = Nsb
+
+    @property
```

Comment:
is it necessary to make these `@property`s?  What's the reasoning behind it as opposed to just attributes?

Reply:
Changed them back to attributes.